continuous-upgrade/jobs/install-ocp.sh

```diff
@@ -21,19 +21,24 @@ cd /data/src/github.com/openshift/aos-cd-jobs/
 git pull origin master
 /data/src/github.com/openshift/aos-cd-jobs/continuous-upgrade/actions/install_junit.sh
 sudo yum-config-manager --disable rhel-7-server-ose-3\*,li
+if [[ -s "${playbook_base}/deploy_cluster.yml" ]]; then
+    playbook="${playbook_base}deploy_cluster.yml"
```

Comment:
missing a / here?

Reply:
Mainly asking because the conditional has a / I don't know if playbook_base ends in a /

Reply:
playbook_base does end in a '/'.  I'll fix these.